### ISSUE TITLE
XeroxPrintDriver.download.recipe is returning a curl error

### ISSUE THREAD

**Issue by flammable** [autopkg/MLBZ521-recipes#37]
Hello! I'm unable to get the XeroxPrintDriver.download recipe working - here's the full output.

Any idea what's wrong? Did Xerox change their website again?

```
% autopkg run -v com.github.mlbz521.download.XeroxPrintDriver
Processing com.github.mlbz521.download.XeroxPrintDriver...
WARNING: com.github.mlbz521.download.XeroxPrintDriver is missing trust info and FAIL_RECIPES_WITHOUT_TRUST_INFO is not set. Proceeding...
XeroxPrintDriverProcessor
XeroxPrintDriverProcessor: Searching for:  Workcentre 6515
XeroxPrintDriverProcessor: Download URL: #
URLDownloader
Command '['/usr/bin/curl', '--silent', '--show-error', '--no-buffer', '--dump-header', '-', '--speed-time', '30', '--location', '--url', '#', '--fail', '--output', '/Users/sjuadmin/Library/AutoPkg/Cache/com.github.mlbz521.download.XeroxPrintDriver/downloads/tmpv7adqnts']' returned non-zero exit status 3.
Failed.
Receipt written to /Users/sjuadmin/Library/AutoPkg/Cache/com.github.mlbz521.download.XeroxPrintDriver/receipts/com.github.mlbz521.download-receipt-20210716-162650.plist

The following recipes failed:
    com.github.mlbz521.download.XeroxPrintDriver
        Error in com.github.mlbz521.download.XeroxPrintDriver: Processor: URLDownloader: Error: Command '['/usr/bin/curl', '--silent', '--show-error', '--no-buffer', '--dump-header', '-', '--speed-time', '30', '--location', '--url', '#', '--fail', '--output', '/Users/sjuadmin/Library/AutoPkg/Cache/com.github.mlbz521.download.XeroxPrintDriver/downloads/tmpv7adqnts']' returned non-zero exit status 3.

Nothing downloaded, packaged or imported.
```

**Comment by MLBZ521** [comment]
I wouldn't be surprised...  I plan to look into this by next week.

**Comment by MLBZ521** [comment]
This has been resolved in 8e01116.

Thanks for the FYI!

**Comment by flammable** [comment]
Thanks! I'll give it a try with my Munki recipe on Monday!
